docs-src/modules/getting-started/pages/index.adoc

```diff
@@ -1,14 +1,141 @@
-Kaskada queries are simple, declarative and composable providing a more expressive way to query event-based data.
+Kaskada is a unified event processing engine that provides stateful stream processing in a high-level query language designed specifically for reasoning about events in bulk and in real time.
 
-There are several ways to quickly get started with Kaskada.
+This document will show you how to quickly get started using Kaskada.
 
-* xref:getting-started:hello-world-jupyter.adoc[]: How to get up and running with Kaskada inside a Jupyter notebook running locally. 
-This quick tutorial will help you install and run Kaskada from within a Jupyter notebook, load some sample data into Kaskada, use Kaskada queries and Python within Jupyter to explore that data.
-* xref:getting-started:hello-world-cli.adoc[]: How to execute queries from the command line using the Kaskada CLI. 
-This quick tutorial will help you install and run Kaskada from the command line, load some sample data into Kaskada, and use Kaskada queries to explore that data all on the command line.
+[TIP]
+====
+Before jumping in to writing queries in Kaskada, it's a good idea to take a minute understanding "how to think with Kaskada".
+Kaskada is built on simple, powerful abstrctions that may be different than what you're accustomed to.
+
+If you'd prefer to jump right in, scroll down to the xref:#quick-starts[] section
+====
+
+== Thinking with Kaskada's timelines
+
+Kaskada is built on the idea of a _timeline_ - the history of how a value changes over time for a specific entity or group.
+
+[stream_viz,name=basic-sum]
+.Aggregating events as a timeline
+....
+[
+    {
+        "label": "Purchase.amount",
+        "kind": "discrete",
+        "data": [
+            [
+                {"t": 2, "v": "5"},
+                {"t": 5, "v": "2"},
+                {"t": 13, "v": "13"},
+                {"t": 18, "v": "4"}
+            ]
+        ]
+    },
+    {
+        "label": "Purchase.amount | sum()",
+        "kind": "continuous",
+        "data": [
+            [
+                {"t": 2, "v": "5"},
+                {"t": 5, "v": "7"},
+                {"t": 13, "v": "20"},
+                {"t": 18, "v": "24"}
+            ]
+        ]
+    }
+]
+....
+
+Timelines give you declarative queries and the ability to reason about temporal context, time travel, sequencing, time-series, and more.
+Transforming and combining timelines allow you to intuitively express computations over events.
+
+****
+⭢ Read more about timelines in xref:overview:what-is-kaskada.adoc[]
+****
+
+With Kaskada, it's timelines all the way down - every operation starts from timelines and produces timelines. 
+When you make a query, the resulting timeline is interpreted in one of two ways: as a _history_ or as a _snapshot_. 
```

Comment:
s/interpreted/output?

Eg., interpreted made me think it changes the meaning of the timeline (affects the timeline) rather than is just a way of outputting it.